src/modules/data.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable */
```

Comment:
- [ ] Linters help us in achieving best practices when writing code. They also help in writing clean and consistent code throughout the codebase. Kindly enable them by removing this line. 😎️